frontend/src/components/accountPage/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+
+export interface IUserInfoProps {
+  fullName: string;
+  username: string;
+  email: string;
+  studentID: string;
+}
+
+const UserInfo: FC<IUserInfoProps> = props => {
+  const { fullName, username, email, studentID } = props;
+  return (
```

Comment:
In this component I would not include the concept of `University`, show the following fields:
- username (which in case of PoliTO students is the matricola)
- email
- first name
- last name